c_src/picosat_nif.c

```diff
@@ -46,14 +44,16 @@ static ERL_NIF_TERM solve_nif(ErlNifEnv* env, int argc, const ERL_NIF_TERM argv[
     return enif_make_atom(env, "unknown");
   } else if (res == PICOSAT_SATISFIABLE) {
     unsigned int vars = picosat_variables(sat);
-    ERL_NIF_TERM solution[vars];
```

Comment:
Visual studio compiler does not support VLA, and apparently never will, reference: https://docs.microsoft.com/en-us/cpp/build/reference/std-specify-language-standard-version?view=vs-2019#c-standards-support-1 (last paragraph in section)